### ISSUE TITLE
Cannot run the example project.

### ISSUE THREAD

**Issue by SametSahin10** [Innim/flutter_login_facebook#39]
I've tried to clone the package and run the example project but I've failed to do so. Below is the error I get:

```
Xcode's output:
↳
    /Users/sametsahin/work/flutter-projects/flutter-mobile-projects/flutter_login_facebook/ios/Classes/SwiftFlutterLoginFacebookPlugin.swift:178:61: error:
    type 'Profile' has no member 'PictureMode'
                    let url = profile.imageURL(forMode: Profile.PictureMode.normal,
                                                        ~~~~~~~ ^~~~~~~~~~~
    /Users/sametsahin/work/flutter-projects/flutter-mobile-projects/flutter_login_facebook/ios/Classes/SwiftFlutterLoginFacebookPlugin.swift:199:11: error:
    extra trailing closure passed in call
            ) { res in
              ^~~~~~~~
    /Users/sametsahin/work/flutter-projects/flutter-mobile-projects/flutter_login_facebook/ios/Classes/SwiftFlutterLoginFacebookPlugin.swift:197:26: error:
    cannot convert value of type '[Permission]' to expected argument type '[String]'
                permissions: permissions,
                             ^
    /Users/sametsahin/work/flutter-projects/flutter-mobile-projects/flutter_login_facebook/ios/Classes/SwiftFlutterLoginFacebookPlugin.swift:197:26: note:
    arguments to generic parameter 'Element' ('Permission' and 'String') are expected to be equal
                permissions: permissions,
                             ^
    /Users/sametsahin/work/flutter-projects/flutter-mobile-projects/flutter_login_facebook/ios/Classes/SwiftFlutterLoginFacebookPlugin.swift:243:34: error:
    value of type 'AccessToken' has no member 'permissions'
                "permissions": token.permissions.map {item in item.name},
                               ~~~~~ ^~~~~~~~~~~
    /Users/sametsahin/work/flutter-projects/flutter-mobile-projects/flutter_login_facebook/ios/Classes/SwiftFlutterLoginFacebookPlugin.swift:244:42: error:
    value of type 'AccessToken' has no member 'declinedPermissions'
                "declinedPermissions": token.declinedPermissions.map {item in item.name},
                                       ~~~~~ ^~~~~~~~~~~~~~~~~~~
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99.
    (in target 'Flutter' from project 'Pods')

Could not build the precompiled application for the device.

Error launching application on Elif iPhone’u.
```

Output of `flutter doctor -v`

```
[✓] Flutter (Channel stable, 1.22.5, on macOS 11.0.1 20B50 darwin-x64, locale en)
    • Flutter version 1.22.5 at /Users/sametsahin/work/flutter
    • Framework revision 7891006299 (4 days ago), 2020-12-10 11:54:40 -0800
    • Engine revision ae90085a84
    • Dart version 2.10.4


[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/sametsahin/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.2, Build version 12B45b
    • CocoaPods version 1.10.0

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[!] IntelliJ IDEA Community Edition (version 2020.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.51.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.16.0
```

**Comment by greymag** [comment]
Hey!

I'm not sure what is the reason of this error. I just cloned the repository and run the example - it's all working.

Do you have this error when launch on simulator or device only? 

Try to remove `/example/ios/Podfile` and run again, something change?

**Comment by SametSahin10** [comment]
This happens both when I try to run the app on Simulator and a physical device. Also the `flutter build ios` command fails too. My coworker has the repository locally and he doesn't have any problems running the project. The environment on my computer and his is almost the exact same. Same Flutter version, Xcode version, packages etc. There's just one thing, I get

`[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries:
    (/Users/sametsahin/work/flutter-projects/flutter-mobile-projects/flambu/ios/Flutter/Flutter.framework)`
    
if I do not comment out `use_frameworks!` on Podfile. After commenting it out, I get the errors related to `flutter_login_facebook` package. I've tried almost every possible solution I've found on the internet. Deleting Podfile, Podfile.lock, deleting the entire ios directory and recreating it, deleting flutter and reinstalling it, switching to `beta` and switching back to `stable` and so on. What could be the reason behind this?
